database/database.go

```diff
@@ -163,6 +163,24 @@ func (db *Database) DeleteRequestToJoinTeam(teamUUID uuid.UUID, fingerprint fpr.
 	return db.saveToFile(*message)
 }
 
+// GetRequestsToJoinTeam returns a slice of requests to join the team matching the UUID.
+func (db *Database) GetRequestsToJoinTeam(teamUUID uuid.UUID) (
```

Comment:
What's the logic behind it returning *multiple* when we actually want to de-duplicate and only get the *earliest* ?

Reply:
Looking at the other PR it seems we *actually* want a function that just queries if a request already exists, rather than actually getting a list of them, right?

Reply:
You're right, see https://github.com/fluidkeys/fluidkeys/pull/405